### DISCUSSION TITLE
I bought a turing 3.5 inch screen and can not get the software to run i feel so stupied<title>

### DISCUSSION THREAD

**Discussion by baddog710** [community/community#68933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i recently bought a new little 3.5 inch display screen to watch my temps ect i got the device and the screen says to go to  www.turzx.com
and run program i followed the steps but when i run config  it won't load so coments diricted me to here what to do  don mount

**Comment by joywin2003** [comment]
watch "temps ect ", didnt they mention it in the manual?

**Reply by baddog710** [reply]
there is 2 differt websites but people seem to be able to run python or git to make display work widows sees it but i can't find a way to download screens temp ect

**Reply by baddog710** [reply]
its the software

**Comment by baddog710** [comment]
the screen just says go to www.turzx.com and it is there and all but when i run config it wont load
